docs/data/data-grid/editing/FullFeaturedCrudGrid.js

```diff
@@ -102,24 +102,20 @@ export default function FullFeaturedCrudGrid() {
     event.defaultMuiPrevented = true;
   };
 
-  const handleEditClick = (id) => (event) => {
-    event.stopPropagation();
```

Comment:
Any `event.stopPropagation()` call prevents the grid from updating the internal state.